### PR TITLE
Feature/uxmcloud 1566: cloud at-internet

### PR DESCRIPTION
**Pull request by whispyy**
Added :
 - event that count the cloud project's number
 - added a trackClick on cloud activation

### CODE REVIEW

File: client/app/cloud/project/add/cloud-project-add.config.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+"use strict";
```
Comment (Mboulianne) [inline client/app/cloud/project/add/cloud-project-add.config.js]: useless use strict.

File: client/app/cloud/project/add/cloud-project-add.config.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+/**
+ *  Configuration for at-internet tracking
+ **/
```
Comment (Mboulianne) [inline client/app/cloud/project/add/cloud-project-add.config.js]: useless comment IMO.

File: client/app/cloud/project/add/cloud-project-add.config.spec.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+"use strict";
```
Comment (Mboulianne) [inline client/app/cloud/project/add/cloud-project-add.config.spec.js]: useless use strict.

File: client/app/cloud/project/add/cloud-project-add.config.spec.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+describe("Controller config: CloudProjectAddCtrl", function () {
```
Comment (Mboulianne) [inline client/app/cloud/project/add/cloud-project-add.config.spec.js]: Implementation is in ES6 but tests are not... that's weird T_T

File: client/app/cloud/project/compute/infrastructure/cloud-project-compute-infrastructure.config.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+"use strict";
+/**
+ *  Configuration for at-internet tracking
+ **/
```
Comment (Mboulianne) [inline client/app/cloud/project/compute/infrastructure/cloud-project-compute-infrastructure.config.js]: Useless comment.

File: client/app/cloud/project/compute/infrastructure/cloud-project-compute-infrastructure.config.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+"use strict";
```
Comment (Mboulianne) [inline client/app/cloud/project/compute/infrastructure/cloud-project-compute-infrastructure.config.js]: Useless use strict.

File: client/app/cloud/project/add/cloud-project-add.config.spec.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+            controller.data = { agreements: [{}] };
+
+            controller.createProject();
+            $httpBackend.flush();
+
+            expect(atInternet.trackClick).toHaveBeenCalledWith({
+                name: "AccountActivation"
+            });
+        }));
+    });
```
Comment (Mboulianne) [inline client/app/cloud/project/add/cloud-project-add.config.spec.js]: missing line break

Comment (whispyy) [reply client/app/cloud/project/add/cloud-project-add.config.spec.js]: Done.

File: client/app/cloud/project/compute/infrastructure/cloud-project-compute-infrastructure.config.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
+angular.module("managerApp")
+    .config(atInternetControllerDecoratorsProvider => {
+        atInternetControllerDecoratorsProvider.decorate({
+            CloudProjectComputeInfrastructureCtrl: {
+                initInfra (atInternet, controller) {
```
Comment (Mboulianne) [inline client/app/cloud/project/compute/infrastructure/cloud-project-compute-infrastructure.config.js]: I don't see a workaround so I approved anyway but that nesting is killing me.

Comment (whispyy) [reply client/app/cloud/project/compute/infrastructure/cloud-project-compute-infrastructure.config.js]: I used to keep consistency with the existing model.
But that part could be review in the futur, and we'll be able to make all changes in one time.

**Pull request merged**